python/lsst/pipe/tasks/calibrateImage.py

```diff
@@ -717,13 +717,14 @@ def _match_psf_stars(self, psf_stars, stars):
         # Closest matches is a dict of psf_stars source ID to Match record
         # (psf_stars source, sourceCat source, distance in pixels).
         best = {}
-        for match0, match1, d in matches:
-            id0 = match0.getId()
+        for match_psf, match_stars, d in matches:
+            id0 = match_psf.getId()
```

Comment:
With the renaming, and since there is no `id1`, I'd suggest just calling this `id` here.